internal/controller/pkg/revision/runtime_test.go

```diff
@@ -404,6 +458,83 @@ func TestRuntimeManifestBuilderDeployment(t *testing.T) {
 	}
 }
 
+func TestRuntimeManifestBuilderService(t *testing.T) {
+	type args struct {
+		builder            ManifestBuilder
+		overrides          []ServiceOverride
+		serviceAccountName string
+	}
+	type want struct {
+		want *corev1.Service
+	}
+	cases := map[string]struct {
+		reason string
+		args   args
+		want   want
+	}{
+		"ProviderServiceNoRuntimeConfig": {
+			reason: "No overrides should result in a deployment with default values",
```

Comment:
this test case is slightly confusing for me because it says "no overrides", but still does pass overrides below, and it also seems to be mentioning provider things but then passing an service override for gRPC which is a function thing.

Any clean-up you think you should do for this unit test?